types/jws/index.d.ts

```diff
@@ -110,7 +110,7 @@ export interface VerifyStream extends events.EventEmitter {
 
 export interface Signature {
     header: Header;
-    payload: any;
+    payload: string;
```

Comment:
I'm unsure about this one - same as above - are you 100% sure this is always a string?  Point me towards the source?